workers/generic-worker/process/multiuser_windows.go

```diff
@@ -172,11 +180,11 @@ func GrantSIDWinstaAccess(sid string, pd *PlatformData) {
 		log.Printf("SID %v NOT found in %#v - granting access...", sid, sidsThatCanControlDesktopAndWindowsStation)
 
 		cmd, err := NewCommand([]string{gwruntime.GenericWorkerBinary(), "grant-winsta-access", "--sid", sid}, ".", []string{}, pd)
-		cmd.DirectOutput(os.Stdout)
-		log.Printf("About to run command: %#v", *(cmd.Cmd))
 		if err != nil {
 			panic(err)
 		}
+		cmd.DirectOutput(os.Stdout)
+		log.Printf("About to run command: %#v", *(cmd.Cmd))
```

Comment:
Nice, thanks for moving these down. Helps with readability with that error handling 👍🏻